### ISSUE TITLE
Customize the `diagnostics_port`

### ISSUE THREAD

**Issue by jbusecke** [dask/dask-jobqueue#241]
I am trying to deploy dask.distributed on Princetons HPC 'tiger'.
Most of dask_jobqueue works great. Thanks for this amazing tool.

I am however not able to set the port for the bokeh dashboard correctly.

When setting up a `LocalCluster` I can successfully pass `diagnostics_port=1224` but in my current try to pass this to `SLURMCluster` I end up with an error:
```
cluster = SLURMCluster(
    job_extra=['-n 2'],
    job_mem="180GB",
    job_cpu=40,
    cores=40,
    memory="180GB",
    interface="em1",
    walltime="00:30:00",
    **{'diagnostics_port':1224}
)
client = Client(cluster)
```

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-7-417e93ae996f> in <module>
      8     interface="em1",
      9     walltime="00:30:00",
---> 10     **{'diagnostics_port':1224}
     11 )
     12 client = Client(cluster)

~/code/miniconda/envs/euc_dynamics/lib/python3.6/site-packages/dask_jobqueue/slurm.py in __init__(self, queue, project, walltime, job_cpu, job_mem, job_extra, **kwargs)
     67             job_extra = dask.config.get('jobqueue.slurm.job-extra')
     68 
---> 69         super(SLURMCluster, self).__init__(**kwargs)
     70 
     71         # Always ask for only one task

~/code/miniconda/envs/euc_dynamics/lib/python3.6/site-packages/dask_jobqueue/core.py in __init__(self, name, cores, memory, processes, interface, death_timeout, local_directory, extra, env_extra, log_directory, walltime, threads, python, **kwargs)
    228         diagnostics_ip_and_port = ('', 8787)
    229         self.local_cluster = LocalCluster(n_workers=0, diagnostics_port=diagnostics_ip_and_port,
--> 230                                           **kwargs)
    231 
    232         # Keep information on process, cores, and memory, for use in subclasses

TypeError: type object got multiple values for keyword argument 'diagnostics_port'
```

It seems that the optional input is internally fixed to 8787? 
Or am I approaching this in the wrong way?

**Comment by guillaumeeb** [comment]
Hi, thanks for raising the issue.

This should be fixed on master branch by #192. But not released for now. Can you try installing from master branch ?

We're planning a release soon, probably early march.

**Comment by jbusecke** [comment]
Thanks a lot that did solve the problem!
